src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -421,6 +429,69 @@ public function getXEditableChoices(FieldDescriptionInterface $fieldDescription)
         return $xEditableChoices;
     }
 
+    /**
+     * Returns a canonicalized locale for "moment" NPM library,
+     * or `null` if the locale's language is "en", which doesn't require localization.
+     *
+     * @param array $context
+     *
+     * @return null|string
+     */
+    public function getCanonicalizedLocaleForMoment(array $context)
+    {
+        $locale = strtolower(str_replace('_', '-', $context['app']->getRequest()->getLocale()));
+
+        // "en" language doesn't require localization.
+        if ('en' === $lang = substr($locale, 0, 2)) {
+            return null;
+        }
+
+        // `moment: ^2.8` only ships "es" and "es-do" locales for "es" language
```

Comment:
Why use backticks here?

Reply:
Beacuse this is a reference to a version constraint declared in [`bower.json` at `sonata-project/core-bundle`](
https://github.com/sonata-project/SonataCoreBundle/blob/63c7b79d521bb084376803babc42f82264113075/bower.json#L11).

Reply:
I mean why use this over quotes, etc... markdown is not recognized here, or is it some other markup? For what tool are you using this markup

Reply:
This is not for a tool, I use backticks just to make explicit that this constraint is used here as a snippet.

Reply:
Ok, why not